mscore/scoreaccessibility.h

```diff
@@ -39,7 +39,7 @@ class ScoreAccessibility : public QObject {
       QMainWindow* mainWindow;
       QLabel* statusBarLabel;
       ScoreAccessibility(QMainWindow* statusBar);
-      std::pair<int, float>barbeat(Element* e);
+      std::tuple<int, int, float>barbeat(Element *e);
```

Comment:
MuseScore code style is `T* t` rather than `T *t`, for historical reasons. Sorry if it's a change from the usual conventions, but it'd be good if you could change the `Element *e` back to `Element* e`.

Reply:
That was my mistake, and I usually do T* t for pointers too. I think the reason in this case was because the signature in scoreaccessibility.cpp originally had T *t, and when I modified the return type I copied that to the header file. Fixed that too now.